.github/workflows/pkg.pr.new-comment.yml

```diff
@@ -0,0 +1,107 @@
+        with:
+          name: output.json
+          path: output.json
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          run-id: ${{ github.event.workflow_run.id }}
+      - name: 'Post or update comment'
+        uses: actions/github-script@v6
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            const fs = require('fs');
+            const output = JSON.parse(fs.readFileSync('output.json', 'utf8'));
+
+            const bot_comment_identifier = `<!-- pkg.pr.new comment -->`;
+
+            const body = (number) => `${bot_comment_identifier}
+
+            [Playground](https://svelte.dev/playground?version=pr-${number})
+
+            \`\`\`
+            ${output.packages.map((p) => `pnpm add https://pkg.pr.new/${p.name}@${number}`).join('\n')}
+            \`\`\`
+            `;
+
+            async function find_bot_comment(issue_number) {
+              if (!issue_number) return null;
+              const comments = await github.rest.issues.listComments({
+                owner: context.repo.owner,
+                repo: context.repo.repo,
+                issue_number: issue_number,
+              });
+              return comments.data.find((comment) =>
+                comment.body.includes(bot_comment_identifier)
+              );
+            }
+
+            async function create_or_update_comment(issue_number) {
+              if (!issue_number) {
+                console.log('No issue number provided. Cannot post or update comment.');
+                return;
+              }
+
+              const existing_comment = await find_bot_comment(issue_number);
+              if (existing_comment) {
+                await github.rest.issues.updateComment({
+                  owner: context.repo.owner,
+                  repo: context.repo.repo,
+                  comment_id: existing_comment.id,
+                  body: body(issue_number),
+                });
+              } else {
+                await github.rest.issues.createComment({
+                  issue_number: issue_number,
+                  owner: context.repo.owner,
+                  repo: context.repo.repo,
+                  body: body(issue_number),
+                });
+              }
+            }
+
+            async function log_publish_info() {
+              console.log('\n' + '='.repeat(50));
+              console.log('Publish Information');
+              console.log('='.repeat(50));
+              console.log('\nPublished Packages:');
+              console.log(packages);
+              console.log('\nPlayground URL:');
+              console.log(`\nhttps://svelte.dev/playground?version=commit-${output.sha.substring(0, 7)}`)
+              console.log('\n' + '='.repeat(50));
+            }
+
+            if (output.event_name === 'pull_request') {
+              if (context.issue.number) {
+                await create_or_update_comment(context.issue.number);
```

Comment:
@paoloricciuti This

Reply:
Actually it will never even run looking at this. None of these events types match.

Reply:
The event name is on the output not on the context...but yeah this I missed it...will fix

Reply:
Ah I see. Makes sense.